ansible/roles/ooni-backend/templates/update_bouncer.py.j2

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+import yaml
+import json
+import urllib
+
+MLAB_NS_QUERY_URL = "{{ mlab_ns_query_url_ansible }}"
```

Comment:
I can change that but I don't really see the reason to do this since we will need to update the task whereas now we need to change only an inventory variable.